_health-care/_js/_form.js

```diff
@@ -18,17 +18,60 @@ var HealthApp = (function() {
     var dateString = "";
 
     if (date.getMonth() < 10) {
-      dateString += "0"
+      dateString += "0";
     }
     dateString += (date.getMonth() + 1) + "/";
 
     if (date.getDate() < 10) {
-      dateString += "0"
+      dateString += "0";
     }
     dateString += date.getDate() + "/";
 
     return dateString + date.getFullYear();
   }
+  function extractMonetaryValueTransform(formData, arg) {
+    return Number(extractTransform(formData, arg)).toFixed(2).toString();
+  }
+  function validatePresence(input) {
+    var valid = true;
```

Comment:
I agree, that reads a lot more clearly! I will change it. Also, I'm not aware of another way to check for the presence of a value (other than `length === 0`). I will look Google around to see if there's an alternative.